### PR TITLE
Allow to edit comments in place

### PR DESCRIPTION
**Pull request by gabrielpoca**
Allows to edit a comment in place instead of scrolling to the new comment form.
https://github.com/groupbuddies/council/issues/140

It doesn't use the sliding comment on mobile, I'll add an issue for that. But I tested and it still usable on mobile.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -37,10 +37,8 @@
 
   create_table "notifications", force: :cascade do |t|
     t.integer "user_id"
-    t.string  "kind"
```
Comment (gabrielpoca) [inline db/schema.rb]: This changes are right, the last migration removes this fields, but the schema was not updated.

**Comment by joaojusto** [comment]
@gabrielpoca this is not working

File: app/assets/javascripts/app/components/comment_card/comment_card.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ctrl.edit = edit;
+    ctrl.update = update;
+
+    function edit() {
+      $element.addClass('is-editing');
+      $element.find('textarea').focus();
+    }
+
+    function update() {
+      $element.removeClass('is-editing');
```
Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_card/comment_card.js]: This only hides the editing form, doesn't updates the data

File: app/assets/javascripts/app/components/comment_card/comment_card.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    function update() {
+      ctrl.disabled = true;
+
+      Comment.update(ctrl.comment.id, {body: ctrl.input})
+        .then(updateComment)
+        .then(stopEditing)
+        .catch(onUpdateError);
+    }
+
+    function startEditing() {
```
Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_card/comment_card.js]: Shouldn't this hide the new comment form also?

Comment (gabrielpoca) [inline app/assets/javascripts/app/components/comment_card/comment_card.js]: Why is that?

Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_card/comment_card.js]: It's a bit odd to see two forms side by side don't you think?

Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_card/comment_card.js]: This is a design question, if its ok then you can merge the code

**Comment by joaojusto** [comment]
:shipit:

**Pull request merged**